platform/ui/src/components/measurementTable/MeasurementTable.js

```diff
@@ -184,11 +196,48 @@ class MeasurementTable extends Component {
   };
 
   getTimepointsHeader = () => {
-    return this.props.timepoints.map((timepoint, index) => {
+    const { timepoints, t, lesionTracker } = this.props;
+
+    return timepoints.map((timepoint, index) => {
       return (
         <div key={index} className="measurementTableHeaderItem">
-          <div className="timepointLabel">{this.props.t(timepoint.key)}</div>
+          <div className="timepointLabel">{t(timepoint.key)}</div>
           <div className="timepointDate">{timepoint.date}</div>
+          {lesionTracker && timepoints.length > 1 && index === 0 && (
+            <div className="caseProgressContainer">
+              <form className="caseProgress" data-key="">
+                <div className="radialProgress">
```

Comment:
Any reason we can't use our built-in svg icon component? If it doesn't work here, can we extract this as its own tiny component so half of the lines of the component aren't an SVG? (easier to grok semantic structure at a glance)